src/components/ResourceListItem.tsx

```diff
@@ -80,6 +83,7 @@ const ResourceListItem: FC<ResourceListItemProps> = ({
     <StyledListItem data-testid={`list-item-resources-${resource.identifier}`}>
       <StyledLinkButton component="a" href={`/resource/${resource.identifier}`}>
         <Thumbnail
+          institution={resource.features.dlr_storage_id ?? institution}
```

Comment:
vi viser institusjonen til innlogget bruker i trefflista om den ikke finner institusjon på ressursen ?  Den skjønte jeg ikke helt ?

Reply:
Enig at dette ikke er optimalt, men ressurs lista som genereres når man spør backend om å få alle sine egne ressurser svarer ikke med hvilken institusjon ressursene har. 
Så dette er gjort som en nødløsning, foreløpig brukes kun ResourceListItem for å vise frem egne ressurser, så da går det med et nødskrik

Reply:
vi kommer vel til å bytte ut ResourceListItem. - så da må vi bare huske på det

Reply:
Endret til at componenten kan ta inn "fallbackInstitution", så er det komponenten som sitter over som bestemmer om brukerens institusjon skal sendes inn eller ikke